test/org/sagebionetworks/bridge/models/surveys/SurveyRuleTest.java

```diff
@@ -94,4 +95,58 @@ public void canSerializeAssignDataGroupRule() throws Exception {
         SurveyRule deser = BridgeObjectMapper.get().treeToValue(node, SurveyRule.class);
         assertEquals(dataGroupRule, deser);
     }
+    
+    @Test
+    public void canSerializeDisplayIf() throws Exception {
+        SurveyRule displayIf = new SurveyRule.Builder().withOperator(Operator.ANY)
+                .withDataGroups(Sets.newHashSet("foo")).withDisplayIf(Boolean.TRUE).build();
+        
+        JsonNode node = BridgeObjectMapper.get().valueToTree(displayIf);
+        assertEquals("any", node.get("operator").asText());
+        assertEquals("foo", node.get("dataGroups").get(0).asText());
+        assertTrue(node.get("displayIf").asBoolean());
```

Comment:
asBoolean() will attempt to parse the JSON value into a boolean. This means we don't know if the serializer is serializing the value as, for example, the int 1 or the string "true". Recommend using booleanValue() instead. (Likewise textValue() instead of asText(), although this matters less often.)

Similarly for canSerializeDisplayUnless().